.github/workflows/tests.yaml

```
@@ -20,15 +20,23 @@ jobs:
       - name: Install framework to local repository
-        run: ./mvnw clean install -DskipTests -f framework/pom.xml --no-transfer-progress
+        run: ./mvnw clean install -DskipTests -f framework/pom.xml --no-transfer-progress -T 1C
       # Build and test remaining artifacts (including examples that depend on framework)
       - name: Build
-        run: ./mvnw --batch-mode -DskipTests package --no-transfer-progress
+        run: ./mvnw --batch-mode -DskipTests package --no-transfer-progress -T 1C
       - name: Verify
-        run: ./mvnw -B verify -P coverage --no-transfer-progress
+        run: ./mvnw -B verify -P coverage --no-transfer-progress -T 1C
       - name: Coverage
         uses: codecov/codecov-action@v3
```

actionlint (1.7.8)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)